pkg/rpm/postupgrade

```diff
@@ -127,66 +127,84 @@ fi
 
 touch /etc/cron.d/cron_zenossdbpack
 
-echo "Creating a backup of elasticsearch-serviced data in $HOST_ISVCS_DIR"
-cp -ar $HOST_ISVCS_DIR/elasticsearch-serviced $HOST_ISVCS_DIR/elasticsearch-serviced.backup
-
 echo "Creating a backup of zookeeper data in $HOST_ISVCS_DIR"
 cp -ar $HOST_ISVCS_DIR/zookeeper $HOST_ISVCS_DIR/zookeeper.backup
 
-echo "Starting docker container elasticsearch-serviced ..."
-docker run --rm -d --name $SVC_NAME -p 9200:9200 \
-  -v $HOST_ES_DATA_DIR:$CONTAINER_ES_DATA_DIR zenoss/serviced-isvcs:v63 \
-  sh -c "$ELASTIC_BIN -f -Des.node.name=$NODE_NAME \
-  -Des.cluster.name=$CLUSTER_NAME"
-
-if check_elasticsearch; then
-    report "SUCCESS" "Container started within timeout"
-else
-    report "FAILURE" "Container failed to start within 120 seconds"
-fi
-
-if migrate_elasticsearch; then
-    report "SUCCESS" "Export completed"
+#
+# CC-4419: If Elasticsearch configuration file and data folder does not exist
+# on the delegate that migration shouldn't start
+#
+if [[ ! -d $HOST_ES_DATA_DIR ] || [ -z $CLUSTER_NAME ]]; then
```

Comment:
This has extra square braces. The form should be [[ ... || ... ]] not [[ ... ] || [ ... ]].